src/mw-list/directives/mw_list_url_action_button_test.js

```diff
@@ -0,0 +1,42 @@
+describe('mwListUrlActionButton', function() {
+
+  beforeEach(function() {
+    module('karmaDirectiveTemplates');
+    module('mwUI');
+  });
+
+  beforeEach(inject(function($compile, $rootScope) {
+    this.$compile = $compile;
+    this.scope = $rootScope.$new();
+  }));
+
+  describe('default behaviour', function() {
+    var compiledElement;
+    beforeEach(function() {
+      compiledElement = this.$compile('<mw-listable-bb><span mw-listable-link-show-bb="http://blog.mwaysolutions.com/"></span></mw-listable-bb>')(this.scope);
+    });
+
+    it('contains the link', function() {
+      this.scope.$digest();
+
+      expect(compiledElement.html()).toContain('href="http://blog.mwaysolutions.com/"');
+    });
+
+    it('does not open the link in a new tab', function() {
+      this.scope.$digest();
+
+      expect(compiledElement.html()).not.toContain('target="_blank"');
```

Comment:
You can also add the attribute in the controller. In case that there is a target you can set the target attribute otherwise not. 
like
```
if(scope.target){
 el.attr('target',scope.target)
}
```